actions/checkForAndPlaceOrder.ts

```diff
@@ -148,6 +155,24 @@ async function _processConditionalOrder(
       }
     }
 
+    // Do custom Conditional Order checks
+    const [handler, salt, staticInput] = await (() => {
+      const { handler, salt, staticInput } = conditionalOrder.params;
+      return Promise.all([handler, salt, staticInput]);
+    })();
+    const validateResult = await validateOrder({
```

Comment:
We validate the static input, if the handler is know. 

This action doesn't care about the specific validation, only the result of it